### ISSUE TITLE
Failed to Activate After Latest Update

### ISSUE THREAD

**Issue by JamesCoyle** [cakecatz/flex-toolbar#131]
[Enter steps to reproduce:]

1. Install latest update
2. Restart Atom

**Atom**: 1.23.3 x64
**Electron**: 1.6.15
**OS**: Microsoft Windows 7 Professional 
**Thrown From**: [flex-tool-bar](https://github.com/cakecatz/flex-toolbar) package 0.15.0

### Stack Trace

Failed to activate the flex-tool-bar package

```
At Cannot read property 'name' of undefined

TypeError: Cannot read property 'name' of undefined
    at Object.storeGrammar (/packages/flex-tool-bar/lib/flex-tool-bar.coffee:363:36)
    at Object.activate (/packages/flex-tool-bar/lib/flex-tool-bar.coffee:50:6)
    at Package.module.exports.Package.activateNow (~/AppData/Local/atom/app-1.23.3/resources/app/src/package.js:255:25)
    at ~/AppData/Local/atom/app-1.23.3/resources/app/src/package.js:227:38
    at Package.module.exports.Package.measure (~/AppData/Local/atom/app-1.23.3/resources/app/src/package.js:99:21)
    at ~/AppData/Local/atom/app-1.23.3/resources/app/src/package.js:220:32
    at Package.module.exports.Package.activate (~/AppData/Local/atom/app-1.23.3/resources/app/src/package.js:217:40)
    at PackageManager.activatePackage (~/AppData/Local/atom/app-1.23.3/resources/app/src/package-manager.js:695:42)
    at config.transactAsync (~/AppData/Local/atom/app-1.23.3/resources/app/src/package-manager.js:670:36)
    at Config.module.exports.Config.transactAsync (~/AppData/Local/atom/app-1.23.3/resources/app/src/config.js:367:24)
    at PackageManager.activatePackages (~/AppData/Local/atom/app-1.23.3/resources/app/src/package-manager.js:668:23)
    at PackageManager.activate (~/AppData/Local/atom/app-1.23.3/resources/app/src/package-manager.js:647:50)
    at ~/AppData/Local/atom/app-1.23.3/resources/app/src/atom-environment.js:865:36
```

### Commands

```
```

### Non-Core Packages

```
atom-beautify 0.30.9 
atom-ide-ui 0.7.2 
atom-material-ui 2.1.1 
atom-overtype-mode 0.4.0 
atom-tag-wrapper 0.2.0 
atom-terminal 0.8.0 
autoclose-html 0.23.0 
browser-plus 0.0.98 
chrome-color-picker 0.8.0 
double-tag 1.1.0 
flex-tool-bar 0.15.0 
fonts 3.0.3 
highlight-line 0.12.0 
highlight-selected 0.13.1 
ide-cpp 0.0.3 
ide-css 0.2.0 
ide-html 0.4.1 
ide-json 0.2.0 
ide-php 0.7.4 
ide-typescript 0.7.2 
language-arduino 0.4.3 
minimap 4.29.7 
minimap-bookmarks 0.4.2 
minimap-find-and-replace 4.5.2 
minimap-highlight-selected 4.6.1 
minimap-selection 4.5.0 
open-in-browser 0.5.2 
platformio-ide-terminal 2.8.0 
rectangle-selection 0.2.0 
release-notes 0.53.0 
selection-highlight 0.1.6 
self-closing-tag 0.2.1 
simple-drag-drop-text 0.3.6 
sync-settings 0.8.3 
todo-show 2.1.0 
tool-bar 1.1.4 
valign 1.1.3 
```

**Comment by UziTech** [comment]
this should be fixed in v0.15.1

**Comment by JamesCoyle** [comment]
Yup. Thanks!
